gradio/chat_interface.py

```diff
@@ -338,8 +339,18 @@ def _setup_example_messages(
                     example_message["files"] = example.get("files", [])
                 if example_labels:
                     example_message["display_text"] = example_labels[index]
-                if example_icons:
-                    example_message["icon"] = example_icons[index]
+                if example_icons and multimodal:
+                    example_files = example_message.get("files")
+                    if not example_files:
+                        example_message["icon"] = {
+                            "path": "",
+                            "url": None,
+                            "orig_name": None,
+                            "mime_type": "text",  # for internal use, not a valid mime type
+                            "meta": {"_type": "gradio.FileData"},
+                        }
+                    else:
+                        example_message["icon"] = example_icons[index]
```

Comment:
if I understand correctly, should this be something like this:
```suggestion
                if multimodal:
                    example_files = example_message.get("files")
                    if not example_files:
                      if not not example_icons:
                        example_message["icon"] = {
                            "path": "",
                            "url": None,
                            "orig_name": None,
                            "mime_type": "text",  # for internal use, not a valid mime type
                            "meta": {"_type": "gradio.FileData"},
                        }
                      else:
                        example_message["icon"] = example_icons[index]
```